cfgov/v1/preprocessed/css/molecules/text-introduction.less

```diff
@@ -0,0 +1,60 @@
+/* topdoc
+  name: Theme variables
+  family: cfgov-molecules
+  notes:
+    - "The following color and sizing variables are exposed,
+       allowing you to easily override them before compiling."
+  patterns:
+  - name: Colors
+    codenotes:
+      - |
+        @m-text-introduction_rule
+  tags:
+  - less
+*/
+@m-text-introduction_rule: @gray-50;
```

Comment:
Oh. I see! All the other rules on the site are set with `@block__border-top` but that name doesn't quite fit here.